src/actor/actor_cell.rs

```diff
@@ -257,8 +250,7 @@ impl TmpActorRefFactory for ActorCell {
     }
 
     fn tmp_actor_of<A: ActorFactory>(&self) -> Result<ActorRef<<A as Actor>::Msg>, CreateError> {
```

Comment:
@vlad9486 
you can also remove all related stuff to `pub trait TmpActorRefFactory`, we dont use it